go.mod

```diff
@@ -244,6 +244,9 @@ require (
 replace (
 	cosmossdk.io/api => cosmossdk.io/api v0.3.1
 
+	github.com/CosmWasm/wasmd => github.com/NibiruChain/wasmd v0.44.0-nibiru
+	github.com/cosmos/cosmos-sdk => github.com/NibiruChain/cosmos-sdk v0.47.11-nibiru
```

Comment:
Small amendments to our current pinned versions. These are needed to create deep copies of the multistore in statedb/journal.go

Reply:
Luckily Cosmos SDK v0.50 implements deep copy for us.